### ISSUE TITLE
Starting Ghost with missing active theme breaks Design Settings screen

### ISSUE THREAD

**Issue by ErisDS** [TryGhost/Ghost#15505]
### Issue Summary

If the selected active theme does not exist in the filesystem the Design Settings screen errors and is unusable.

The problem occurs because Ghost's API is returning an unexpected state - when requesting a list of themes none of them has the active: true flag.

The preferred solution is to cater for this state in Admin somehow

e.g. have a blank-slate state for the design settings screen indicating that no theme is active and point user to the change-theme screen

### Steps to Reproduce

1. Activate a custom theme
2. Stop Ghost and delete or rename the custom theme directory
3. Start Ghost
4. Try to access the design settings screen

![image](https://user-images.githubusercontent.com/101513/193050450-fdd1b30c-f923-49a8-99a6-eedb5c5a96e6.png)

### Ghost Version

5.16

### Node.js Version

16

### How did you install Ghost?

ghost install

### Database type

MySQL 8

### Relevant log / error output

```shell
<img width="630" alt="image" src="https://user-images.githubusercontent.com/101513/193117369-f2947482-993a-440e-91c8-6f5cc11a5c37.png">
```

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by Walexero** [comment]
Do i have to have a NodeJS knowledge to work on the issue or having a plain JS knowledge woulod suffice, if it would, i would like to work on the issue.

**Comment by ErisDS** [comment]
Hi @Walexero, this is a bug in our Ember.js admin client. If you understand the code enough to fix the issue, then you have enough knowledge... the only person who knows if you have enough knowledge is you!

**Comment by Walexero** [comment]
Hello @ErisDS, i already pulled ghost and trying to work on it locally, i first starteed the server with `yarn dev` but just gave the home page and didn't provide the theme option, tried signing up locally but doesn't provide options beyond those provided before signing up. Then i stopped the server and ran it with `yarn dev:admin` but the server errored out with deprecation issues. I had ghost set up with the configuration guidelines on https://ghost.org/docs/install/source/[ghost setup guideline] so is there anything wrong with the setup i did?

**Comment by ErisDS** [comment]
Doesn't sound like there was anything wrong with your setup. This is a bug in Ghost's admin panel, which lives at `/ghost/`.

**Comment by Walexero** [comment]
do you have a way to figure this out so i can work on this open issue

**Comment by ErisDS** [comment]
I have updated the [source install docs](https://ghost.org/docs/install/source/) to include the location of Ghost Admin when using `yarn dev`. I am not sure that there is anything else to figure out? 

Beyond framework knowledge there are certain product concepts in Ghost that you need to understand to fix this bug: that Ghost Admin exists and how to access it, and what a theme is and how to use one. This is all covered in our documentation, but Ghost is a large project.

**Comment by Walexero** [comment]
Apart from being able to review the source code for the theme implementation and accessing it through the Ghost Admin to make changes, do you think the code readability would be an issue for me based off the fact you said its a large project and is there a way to get in touch with you whilst making the change implementation

**Comment by kristophesankar** [comment]
Hi @ErisDS, I took a quick look at this issue and came up with this. 

![gh](https://user-images.githubusercontent.com/33190221/194423019-af697e49-dd85-4cd2-8c26-612a514acf2b.png)

I've added a "none selected" to the change theme button and the section in the middle informing the user that there isn't any active theme. I do have one question though. Should the Brand dropdown in the menu be hidden, or disabled maybe? If not  I'll go ahead and submit a PR.
